docs/user-guide/grpc.md

```diff
@@ -8,102 +8,97 @@ Ambassador makes it easy to access your services from outside your application.
 
 This tutorial assumes you have already followed the [Ambassador Getting Started](/user-guide/getting-started.html) guide. If you haven't done that already, you should do that now.
 
-After completing [Getting Started](/user-guide/getting-started.html), you'll have a Kubernetes cluster running Ambassador and the Quote of the Moment service. Let's walk through adding the [Hello World gRPC service](https://github.com/grpc/grpc-go/tree/master/examples/helloworld) for this tutorial. 
+After completing [Getting Started](/user-guide/getting-started.html), you'll have a Kubernetes cluster running Ambassador and the Quote of the Moment service. Let's walk through adding an example [echo gRPC service](https://github.com/datawire/grpc-example) for this tutorial. 
```

Comment:
Is there a reason why you created this fork, instead of using the original example? This adds to maintenance work over time.